locale/fi/toast.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ponnahdusilmoitus (Toast)",
+  "intro": "Ponnahdusilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Ponnahdusilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ponnahdusilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ponnahdusilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ponnahdusilmoituksesta.",
```

Comment:
I'm still not quite sure if this is clear enough. Somehow when I read this line, I get the impression that the live region itself is an external link at the bottom of the page. Not sure if we should omit the link entirely and just assume they can look up what live region means. What do you think?

Reply:
fixed to be:
"(kts. linkki sivun alalaidassa)"